commons/src/main/java/org/infinispan/commons/util/Util.java

```diff
@@ -1099,4 +1102,22 @@ public static String xmlify(String s) {
    public static String[] stringArray(int length) {
       return length == 0 ? EMPTY_STRING_ARRAY : new String[length];
    }
+
+   public static void renameTempFile(File tempFile, File lockFile, File dstFile, BiConsumer<File, File> renameFailed)
+         throws IOException {
+      FileLock lock = null;
+      try (FileOutputStream lockFileOS = new FileOutputStream(lockFile)) {
+         lock = lockFileOS.getChannel().lock();
+         if (!tempFile.renameTo(dstFile)) {
+            renameFailed.accept(tempFile, dstFile);
+         }
+      } finally {
+         if (lock != null && lock.isValid()) {
+            lock.release();
+         }
+         if (!lockFile.delete()) {
+            log.debugf("Unable to delete lock file %s", lockFile);
```

Comment:
log debug here is in purpose ?

Reply:
yes, just to remove the yellow background warning that I'm ignoring the `delete()` result.

Reply:
ok